src/frontends/llvm/l2s_instr.c

```diff
@@ -394,11 +394,11 @@ EmittedInstr convert_instruction(Parser* p, Node* fn_or_bb, BodyBuilder* b, LLVM
                     String name = get_string_literal(target->arena, name_node);
                     assert(name);
                     set_variable_name((Node*) target, name);
-                    return (EmittedInstr) {};
+                    //return {};
                 }
                 if (strcmp(intrinsic, "llvm.dbg.label") == 0) {
                     // TODO
-                    return (EmittedInstr) {};
+                    //return {};
```

Comment:
TODO: do not leave it without return, this thing requires attention and msvc doesn't like the previous statement